webui/react/src/components/ResourcePoolCard.module.scss

```diff
@@ -0,0 +1,136 @@
+$padding: var(--theme-sizes-layout-large);
+$icon-side: 5rem;
+$line-height: 2.2rem;
+
+.base {
+  border-radius: var(--theme-sizes-border-radius);
+  box-shadow: 2px 2px 8px rgba(0, 0, 0, 0.15);
+  color: var(--theme-colors-monochrome-0);
+  display: flex;
+  flex-direction: column;
+  font-size: var(--theme-sizes-font-medium);
+  line-height: $line-height;
+  overflow: hidden;
+  text-overflow: ellipsis;
+
+  .body {
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1;
+  }
+  .header {
+    background: #234b65;
```

Comment:
**chore:** my mistake on this color in figma, can you change it to `var(--theme-colors-base-3)`?